source/initial_temperature/adiabatic_boundary.cc

```diff
@@ -87,21 +67,18 @@ namespace aspect
     {
       prm.enter_subsection ("Initial temperature model");
       {
+        Utilities::AsciiDataBase<dim>::declare_parameters(prm,
+                                                          "$ASPECT_SOURCE_DIR/data/initial-temperature/adiabatic-boundary/",
+                                                          "adiabatic_boundary.txt",
+                                                          "Adiabatic boundary");
         prm.enter_subsection("Adiabatic boundary");
         {
-          prm.declare_entry ("Data directory", "$ASPECT_SOURCE_DIR/data/initial-temperature/adiabatic-boundary/",
-                             Patterns::DirectoryName (),
-                             "The path to the isotherm depth data file");
-          prm.declare_entry ("Isotherm depth filename",
-                             "adiabatic_boundary.txt",
-                             Patterns::FileName (),
-                             "File from which the isotherm depth data is read.");
           prm.declare_entry ("Isotherm temperature", "1673.15",
                              Patterns::Double (0),
-                             "The value of the isothermal boundary temperature. Units: $\\si{K}$.");
+                             "The value of the isothermal boundary temperature. Units: Kelvin.");
```

Comment:
Please keep this unless you had a specific reason to change it. If I remember correctly the `\\si` command will make sure to print the unit symbol correctly.